spec/ScorecardSpec.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+describe("Scorecard", function(){
+  var scorecard = new Scorecard();
+
+  describe("constructor", function(){
+    it("creates a new instance", function(){
+      expect(scorecard instanceof Scorecard).toBe(true)
+    })
+  })
+
+  beforeEach(function(){
+    scorecard = new Scorecard();
```

Comment:
Efficient testing by using beforeEach so you do not have to repeat creating a new scorecard for each test.